src/models/order.js

```diff
@@ -1,16 +1,23 @@
-import InternalUtil from '../utils/internal_util';
+import Utils from '../utils/util';
 import EasyPostObject from './easypost_object';
 
+/**
+ * An {@link https://www.easypost.com/docs/api/node#orders Order} represents a collection of packages, intended only for multi-parcel shipments.
+ * @public
+ * @extends EasyPostObject
+ */
 export default class Order extends EasyPostObject {
   /**
-   * Get the lowest rate of an order.
-   * @param {array} carriers
-   * @param {array} services
-   * @returns {Object}
+   * Get the lowest rate for this {@link Order}.
+   * @public
+   * @param {string[]} [carriers] - List of allowed carriers to filter by
+   * @param {string[]} [services] - List of allowed services to filter by
+   * @returns {Rate} - The lowest rate
+   * @throws {FilteringError} - If no applicable rates are found
    */
   lowestRate(carriers, services) {
-    const lowestRate = InternalUtil.getLowestObjectRate(this, carriers, services);
+    const rates = this.rates || [];
 
-    return lowestRate;
+    return Utils.getLowestRate(rates, carriers, services);
```

Comment:
Question: Why was this change made? It shouldn't have been necessary. It raises implementation details up a level

Reply:
I like this way, this allows us to remove the duplicate helper function for stateless rate as we are grabbing the object rate first before passing to the generic `getLowestRate` helper function.